src/pure.ts

```diff
@@ -0,0 +1,31 @@
+import { RenderingEngineArray, ReactHooksRenderer } from 'types'
+
+const RENDERERS: RenderingEngineArray = [
+  { required: 'react-test-renderer', renderer: './native/pure' },
+  { required: 'react-dom', renderer: './server/pure' }
```

Comment:
The default renderer for `react-dom` should be `./dom/pure`.  Server rendering is opt in for testing edge cases.